collect_app/src/main/java/org/odk/collect/android/widgets/interfaces/MediaManagerListener.java

```diff
@@ -0,0 +1,7 @@
+package org.odk.collect.android.widgets.interfaces;
+
+public interface MediaManagerListener {
```

Comment:
Not sure the naming is right here. It's not really a "listener". Maybe the interface is `QuestionMediaManager` and the implementation is something else?

Reply:
Yes right the implementation mostly concerns with the deletion or replacing the file at the current index. But I think I need to be extending this implementation for working on other widgets